### PR TITLE
added words

### PR DESCRIPTION
**Pull request by MaathavanJkr**
Added Words to Dix

### CODE REVIEW

File: apertium-eng.eng.dix [hunk 1]
```diff
@@ -44838,6 +44838,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="Albert of Monaco"><i>Albert<b/>of<b/>Monaco</i><par n="Mark__np"/></e>
 <e lm="Alberto">         <i>Alberto</i><par n="Mark__np"/></e>
 <e lm="Albicio">         <i>Albicio</i><par n="Mark__np"/></e>
+<e lm="Albido">         <i>Albido</i><par n="Mark__np"/></e>
```
Comment (jonorthwash) [inline apertium-eng.eng.dix]: spacing is wrong

File: apertium-eng.eng.dix [hunk 2]
```diff
@@ -48958,6 +48962,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="William">         <i>William</i><par n="Mark__np"/></e>
 <e lm="William Tell">    <i>William<b/>Tell</i><par n="Mark__np"/></e>
 <e lm="Williams">        <i>Williams</i><par n="Mark__np"/></e>
+<e lm="Williamson">      <i>Williamson</i><par n="Mark__np"/></e>
```
Comment (jonorthwash) [inline apertium-eng.eng.dix]: This is a personal name?

File: apertium-eng.eng.dix [hunk 3]
```diff
@@ -65069,133 +65079,412 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 <e lm="Seoul">           <i>Seoul</i><par n="Barcelona__np"/></e>
+<e lm="Sepang">          <i>Sepang</i><par n="Barcelona__np"/></e>
+<e lm="Seraing">         <i>Seraing</i><par n="Barcelona__np"/></e>
+<e lm="Serang">          <i>Serang</i><par n="Barcelona__np"/></e>
 <e lm="Serbia">          <i>Serbia</i><par n="Barcelona__np"/></e>
+<e lm="Sergiyev Posad">  <i>Sergiyev<b/>Posad</i><par n="Barcelona__np"/></e>
+<e lm="Seropedica">      <i>Seropedica</i><par n="Barcelona__np"/></e>
+<e lm="Serpukhov">       <i>Serpukhov</i><par n="Barcelona__np"/></e>
+<e lm="Serra Talhada">   <i>Serra<b/>Talhada</i><par n="Barcelona__np"/></e>
+<e lm="Sete Lagoas">     <i>Sete<b/>Lagoas</i><par n="Barcelona__np"/></e>
```
Comment (jonorthwash) [inline apertium-eng.eng.dix]: Is this categorised right?  Check categories for place names that are compositionally plurals.

File: apertium-eng.eng.dix [hunk 4]
```diff
@@ -65069,133 +65079,412 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+<e lm="South Lawndale">  <i>South<b/>Lawndale</i><par n="Barcelona__np"/></e>
 <e lm="South Pole">      <i>South<b/>Pole</i><par n="Barcelona__np"/></e>
+<e lm="South San Francisco"><i>South<b/>San<b/>Francisco</i><par n="Barcelona__np"/></e>
 <e lm="South Sudan">     <i>South<b/>Sudan</i><par n="Barcelona__np"/></e>
 <e lm="South Wales">     <i>South<b/>Wales</i><par n="Barcelona__np"/></e>
 <e lm="Southampton">     <i>Southampton</i><par n="Barcelona__np"/></e>
 <e lm="Southern Bavaria"><i>Southern<b/>Bavaria</i><par n="Barcelona__np"/></e>
 <e lm="Southern Europe"> <par n="Ss"/><i>outhern<b/>Europe</i><par n="Barcelona__np"/></e>
-<e lm="South-Korea">     <i>South-Korea</i><par n="Barcelona__np"/></e>
-<e lm="south-western Iceland"><i>south-western<b/>Iceland</i><par n="Barcelona__np"/></e>
```
Comment (jonorthwash) [inline apertium-eng.eng.dix]: Why were these removed?  This could break a released language pair.

File: apertium-eng.eng.dix (hunk 2 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: sorry he is cricket player. and its a last name. its a fault i will delete it. :(

File: apertium-eng.eng.dix (hunk 3 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: i cant understand. can u please explain

File: apertium-eng.eng.dix (hunk 4 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: they are not removed i just removed the "-". is it make any changes??

File: apertium-eng.eng.dix (hunk 1 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: i will correct it. :)

File: apertium-eng.eng.dix (hunk 3 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: Yeah that is a city in Brazil, and everything i added in places name are cities! :)

File: apertium-eng.eng.dix (hunk 1 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: done :)

File: apertium-eng.eng.dix (hunk 4 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: line 65439 and 65440 its still there but it says it removed.  and why is that?

Comment (jonorthwash) [reply apertium-eng.eng.dix]: Why does it show that these lines were removed?

File: apertium-eng.eng.dix (hunk 3 above)
Comment (jonorthwash) [reply apertium-eng.eng.dix]: Does it matter that they're cities?  Note the others you added that are plurals too.

Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: So can i remove those are plurals?

File: apertium-eng.eng.dix (hunk 4 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: i dont know too.

Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: I just added it before wait that must be the reason. alphabetical sorting error.
the symbols are after the letters :).

Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: *space

Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: Done :)

File: apertium-eng.eng.dix (hunk 3 above)
Comment (MaathavanJkr) [reply apertium-eng.eng.dix]: okay i got it now!
